### PR TITLE
Adding docs for message bridge

### PR DESCRIPTION
**Pull request by shakyShane**
**Asana Task/Github Issue:** https://app.asana.com/0/1201141132935289/1208785555090562/f

## Description

See the preview link: https://deploy-preview-1230--content-scope-scripts.netlify.app/

image

## Testing Steps

- 

## Checklist

*Please tick all that apply:*

- [ ] I have tested this change locally
- [ ] I have tested this change locally in all supported browsers
- [ ] This change will be visible to users
- [ ] I have added automated tests that cover this change
- [ ] I have ensured the change is gated by config
- [ ] This change was covered by a ship review
- [ ] This change was covered by a tech design
- [ ] Any dependent config has been merged

### CODE REVIEW

**Comment by shakyShane** [comment]
* **#1230** <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1230?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈 <a href="https://app.graphite.dev/github/pr/duckduckgo/content-scope-scripts/1230?utm_source=stack-comment-view-in-graphite" target="_blank">(View in Graphite)</a>
* `main`

<h2></h2>

This stack of pull requests is managed by <a href="https://graphite.dev?utm-source=stack-comment"><b>Graphite</b></a>. Learn more about <a href="https://stacking.dev/?utm_source=stack-comment">stacking</a>.

File: injected/docs/message-bridge.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+**NOTE:** Native platforms should continue to verify the sending domain when any messages are received.  
+
+## Example
+
+In a domain where the bridge is enabled, the following API becomes available to the page.
+
+```javascript
+const bridge = navigator.duckduckgo?.createMessageBridge?.('exampleFeature');
```
Comment (jonathanKingston) [inline injected/docs/message-bridge.md:75]: The last one's not needed right?

```suggestion
const bridge = navigator.duckduckgo?.createMessageBridge('exampleFeature');
```

**Review by jonathanKingston — APPROVED** [review]

File: injected/docs/message-bridge.md (hunk 1 above)
Comment (shakyShane) [reply injected/docs/message-bridge.md:75]: yeah it is, because the SERP will be deployed and accessed on devices that have not got the latest CSS yet.

Comment (jonathanKingston) [reply injected/docs/message-bridge.md:75]: Excellent point :)

Comment (jonathanKingston) [reply injected/docs/message-bridge.md:75]: also :(

**Pull request merged**